docs/source/models/causes/other_causes/index.rst

```diff
@@ -74,16 +74,21 @@ States Data
      - 
    * - C
      - disabilty weight
-     - all_cause_yld_rate - anemia_yld_rate - population_maternal_disorders_yld_rate + pregnancy_maternal_disorders_yld_rate
+     - all_cause_yld_rate - anemia_yld_rate - population_maternal_disorders_yld_rate 
      - See extended note below
 
 .. note::
 
   Note that we are assuming the YLD rate due to all causes other than maternal disorders (including YLDs due to anemia) do not significantly vary between the pregnant/postpartum not pregnant/postpartum populations. We do rescale the rate of maternal disorders from the general population of women of reproductive age to the rate among the pregnant/postpartum population in this equation. 
```

Comment:
The "We do rescale..." sentence is no longer true, I think.

Reply:
Correct! Thanks for catching.